apache-maven/src/assembly/shared/run

```diff
@@ -14,6 +14,7 @@ exec "$JAVACMD" \
   $MAVEN_OPTS \
   $MAVEN_DEBUG_OPTS \
   -classpath "${LAUNCHER_JAR}" \
-  $MAVENHOME_CONFIG \
+  "-Dclassworlds.conf=$CLASSWOLRDS_CONF" \
```

Comment:
typo in `CLASSWOLRDS`, but why use/introduce a variable if we can say `-Dclassworlds.conf=${MAVEN_HOME}/bin/m2.conf` ?

Reply:
Thanks, will fix that.
We cannot add the system property name to the variable value because a subsequent ticket will handle the Cygwin path conversion between Unix and Windows. This is not covered for the moment. If you look at `sh -x mvn ...` you'll see that still the Unix path is passed and we rely on implicit Cygwin behavior.

Reply:
Ok, I see. If the usage of `$CLASSWOLRDS_CONF` fixes that I'm okay with it.
We might need tests for it, otherwise we'll have a future committer that tries to clean up these scripts too, not aware of the reason ;)

Reply:
I have a local patch which handles the path conversion and will provide after this one has been merged. It works both on Cygwin and MinGW.

Reply:
Here are the details: https://www.msys2.org/docs/filesystem-paths/